src/os/bluestore/BlueStore.cc

```diff
@@ -2733,6 +2733,10 @@ void BlueStore::_init_logger()
     "Average finishing state latency");
   b.add_time_avg(l_bluestore_state_done_lat, "state_done_lat",
     "Average done state latency");
+  b.add_time_avg(l_bluestore_ack_lat, "ack_lat",
+    "Average ack latency");
+  b.add_time_avg(l_bluestore_commit_lat, "commit_lat",
+    "Average commit latency");
```

Comment:
No need for 2 of these; ack and commit callbacks are triggered at the same time for bluestore